contributors/devel/cherry-picks.md

```diff
@@ -19,26 +20,55 @@ task is for backporting PRs from master to release branches.
+ * Run the [cherrypick script](https://git.k8s.io/kubernetes/hack/cherry_pick_pull.sh).
    This example applies a master branch PR #98765 to the remote branch
    `upstream/release-3.14`: `hack/cherry_pick_pull.sh upstream/release-3.14
    98765`
+   * Be aware the cherrypick script assumes you have a git remote called `upstream` that points at the Kubernetes github org.
+   It will also expect the following environment variables:
+   ```bash
+   FORK_REMOTE=<remote-of-your-fork> \
+   UPSTREAM_REMOTE=<remote-of-upstream> \
+   GITHUB_USER=<your-github-user> \
+   ./hack/cherry_pick_pull.sh <remote-of-upstream>/<release-x.xx> <master-branch-pr-number>
+   ```
+   * You will need to run the cherrypick script separately for each patch release you want to cherrypick to.
+
  * Your cherrypick PR targeted to the release branch will immediately get the
-   `do-not-merge/cherry-pick-not-approved` label. The release branch owner
+   `do-not-merge/cherry-pick-not-approved` label. The [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager)
    will triage PRs targeted to the branch.  Normal rules apply for code merge.
    * Reviewers `/lgtm` and owners `/approve` as they deem appropriate.
-   * The approving release branch owner is responsible for applying the
-     `cherrypick-approved` label.
+   * Milestone approval needs to be added for the target release branch.
+   This should be the most current milestone.
+   * The [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager) for the target release branch is responsible for
+   applying the `cherrypick-approved` label.
+   You may cc them with `<@githubusername>` on your cherrypick PR.
 
-## Cherry Pick Review
+## Cherrypick Review
 
-Cherry pick pull requests are reviewed differently than normal pull requests. In
-particular, they may be self-merged by the release branch owner without fanfare,
-in the case the release branch owner knows the cherry pick was already
-requested - this should not be the norm, but it may happen.
+Cherrypick pull requests have an additional requirement compared to normal pull
+requests.
+They must be approved specifically for cherrypick by Approvers.
+Additionally, the [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager)
+is the final and sole authority on removing the `do-not-merge/cherry-pick-not-approved`
+label and triggering a merge into the current patch release.
 
-## Searching for Cherry Picks
+## Searching for Cherrypicks
 
 See the [cherrypick queue dashboard](http://cherrypick.k8s.io/#/queue) for
 status of PRs labeled as `cherrypick-candidate`.
+
+## Troubleshooting Cherrypicks
+
+Contributors may encounter some of the following difficulties when initiating a cherrypick.
+
+- A cherrypick PR does not apply cleanly against an old release branch.
+In that case, you will need to manually fix conflicts and rebase.
```

Comment:
Do we need the `and rebase` in the end? I think fixing conflicts is enough. If we rebase, what do we rebase on top of?

Reply:
Yep - too much detail can get confusing. Will take out.